Automattic/jetpack
# Boost: Add consolation log on critical CSS gen

Fixes #21370 

#### Changes proposed in this Pull Request:

Add a console log with the following content right before the Critical CSS generation process:
```
Generating Critical CSS will often leave errors in your console. Most of them are nothing to worry about - just the sounds of your browser complaining about our strict security settings. Check out the following page for more information: {link}
```

#### Jetpack product discussion


p1634020657012900-slack-C0232B8CUHG

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Navigate to Boost dashboard
* Open developer tool in the browser
* Generate Critical CSS
* Verify that the following message is logged in the developer tool console right before any error is logged related to Blocked script execution. Eg:
<img width="1166" alt="consolation-note" src="https://user-images.githubusercontent.com/8727183/149971905-3a040226-b636-4d08-b98c-df04df3c37d9.png">
